File: FRS/FRS/ClientApp/src/app/models/meal-order/email-template.ts

export class EmailTemplate {

  constructor(id?: string, subject?: string) {

    this.id = id;
    this.subject = subject;
  }

  public id: string;
  public subject: string;
  public body: string;
  public fromName: string;
  public fromEmail: string;
  public institutionId: string;
  public outletId: string;
}

File: FRS/DAL/Models/EmailTemplate.cs
using DAL.Models.MealOrder;
using Sieve.Attributes;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace DAL.Models
{
    public class EmailTemplate : AuditableEntity
    {
        [Key]
        public int Id { get; set; }
        public string Subject { get; set; }
        public string Body { get; set; }
        public string FromEmail { get; set; }
        public string FromName { get; set; }
        public int? InstitutionId { get; set; }
        public int? OutletId { get; set; }
        public virtual Outlet Outlet { get; set; }
        public virtual Institution Institution { get; set; }
    }
}
